tests/graph/test_op.py

```diff
@@ -223,3 +223,16 @@ def perform(self, node, inputs, outputs):
     msg = r"^Invalid input types for Op.*"
     with pytest.raises(TypeError, match=msg):
         TestOp()(dvector(), dscalar(), dvector())
+
+
+def test_op_broadcastable():
+    # Test that we can create an op with a subtype as input
+    class SomeOp(aesara.tensor.Op):
+        itypes = [at.dvector]
+        otypes = [at.dvector]
+
+        def perform(self, *_):
+            raise NotImplementedError()
+
+    x = at.dscalar("x")
+    SomeOp()(x[None])
```

Comment:
Perhaps we should assert something about the output, otherwise looks good

Reply:
done